roles/os_hardening/tasks/minimize_access_fs.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: "Configure mount {{ mount.path }} with hardened options"
+  mount:
+    path: "{{ mount.path }}"
+    src: "{{ mount.src | default(mountinfo.device, true) }}"
+    fstype: "{{ mount.fstype | default(mountinfo.fstype, true) }}"
+    opts: "{{ mount.opts }}"
+    dump: "{{ mount.dump | default('1' if mount.fstype | default(mountinfo.fstype, true) in ['ext3', 'ext4'] else '0', true) }}"
+    passno: "{{ mount.passno | default('2' if mount.fstype | default(mountinfo.fstype, true) in ['ext3', 'ext4'] else '0', true) }}"
+    state: present
+  register: mountpoint
+  when:
+    - mount.enabled | bool
+    - mount.path in mountpoints_list
+  vars:
+    mountinfo: "{{ ansible_mounts | selectattr('mount', 'equalto', mount.path) | list | first | default(None) }}"
+  notify: remount fs
```

Comment:
```suggestion
  notify: Remount filesystems
```

Reply:
maybe the notify is better added to the task "Register changed mountpoints", since it depends on the variable constructed there.

Reply:
as far as i know the `notify` can not be added to a `set_fact`, because this does never give a "change" and because of that the notify will not be triggered.
Perhaps it would be possible with a `change_when` added, but that is a ugly hack in my opinion.

Reply:
you are right, if we add the `notify` to the `set_fact` we would need to also add a `changed_when: true`. It would work nicely, since the set_fact is only done when it needs to run, but this would be another layer of magic in this PR. :D

What do you think @rndmh3ro ?

Reply:
The way it is right now is fine.